### DISCUSSION TITLE
Remove ">>" buttons which jumps 1 year ahead

### DISCUSSION THREAD

**Discussion by Preet-Sojitra** [wojtekmaj/react-calendar#713] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/87543808/207034998-bf629c74-6b74-46c1-b711-ba27eda2d9b0.png)
 How to remove these ">>" buttons, I don't want the use to move 1 year forward. 
 Is there any way to disable those?

**Comment by wojtekmaj** [comment, marked as answer]
In README you have:

Prop name | Description
-- | --
next2Label | Content of the "next on higher level" button on the navigation pane. Setting the value explicitly to null will hide the icon.

So,

`next2Label={null}` and there you have it :)

**Comment by MRUNAL727** [comment]
Example - 
   `<Calendar` onChange={handleChnage} value={value} selectRange={true}
        next2Label={null} prev2Label={null}/>`
